src/jsmd-tools.js

```diff
@@ -111,29 +135,48 @@ function parseJsmd(jsmd) {
   return { cells, parseWarnings }
 }
 
-const cellTypeToJsmdMap = new Map([
-  ['javascript', 'js'],
-  ['markdown', 'md'],
-  ['external dependencies', 'resource'],
-  ['raw', 'raw'],
-])
+function stateFromJsmd(jsmdString) {
+  const parsed = parseJsmd(jsmdString)
+  let { cells } = parsed
+  const { parseWarnings } = parsed
+  console.log(parseWarnings)
```

Comment:
let's not log parseWarnings unless this array has some contents, and perhaps we use `console.warn` over the contents.

Reply:
good idea, fixed